### PR TITLE
display data-migration-version in user settings page

### LINKED ISSUE #2976: Display Data Migration Version in prod-release-0-2-5
**Describe the feature request.**
Display Data Migration Version

**Additional context**
Display Data Migration Version

**Area path**
A list of ordered steps in the app on usage of the feature to support anyone testing it e.g. Code reviewer, QA e.g.
1. Login to the app
2. Open _Navigation bar_
3. Click on _Settings_
4. Check Data Migration Version

**Acceptance criteria**
Data Migration Version must appears

### PR DESCRIPTION
**Pull request by AbdulWahabMemon**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes #2976 

**Engineer Checklist**
- [x] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [x] I have added any strings visible on UI components to the `strings.xml` file
- [ ] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [x] I have built and run the FHIRCore app to verify my change fixes the issue and/or does not break the app 
- [ ] I have checked that this PR does NOT introduce **breaking changes** that require an update to **_Content_** and/or **_Configs_**? _If it does add a sample here or a link to exactly what changes need to be made to the content._

**Code Reviewer Checklist**
- [ ] I have verified **Unit tests** have been written for any new feature(s) and edge cases
- [ ] I have verified any strings visible on UI components are in the `strings.xml` file
- [ ] I have verifed the [CHANGELOG.md](./CHANGELOG.md) file has any notable changes to the codebase
- [ ] I have verified the solution has been implemented in a configurable and generic way for reuseable components
- [ ] I have built and run the FHIRCore app to verify the change fixes the issue and/or does not break the app

### CODE REVIEW

File: android/quest/src/androidTest/java/org/smartregister/fhircore/quest/ui/report/measure/components/ReportTypeSelectorScreenTest.kt [hunk 1]
```diff
@@ -165,10 +165,10 @@ class ReportTypeSelectorScreenTest {
         dateRange = null,
         reportPeriodRange = mapOf(),
         onBackPressed = {},
-        onGenerateReport = {},
```
Comment (Rkareko) [inline android/quest/src/androidTest/java/org/smartregister/fhircore/quest/ui/report/measure/components/ReportTypeSelectorScreenTest.kt:168]: Why was this removed?

Comment (AbdulWahabMemon) [reply android/quest/src/androidTest/java/org/smartregister/fhircore/quest/ui/report/measure/components/ReportTypeSelectorScreenTest.kt:168]: it fails while running other tests, that property must have removed

**Review by Rkareko — APPROVED** [review]
